backend/frontend/src/App.js

```diff
@@ -13,11 +13,33 @@ const theme = createTheme({
   },
 });
 
+const customTheme = createTheme({
+  palette: {
+    primary: {
+      main: '#ff9800', // Change this to your custom color
+    },
+    secondary: {
+      main: '#f57c00', // Change this to your custom color
+    },
+    text: {
+
+    },
+  }
+});
+
 function App() {
+  const [currentTheme, setCurrentTheme] = useState('default'); // add a state variable for the current theme
+
+  const toggleTheme = () => { // add a function to toggle the theme
+    setCurrentTheme(currentTheme === 'default' ? 'custom' : 'default');
```

Comment:
I prefer
default -> light
custom -> dark
naming

Reply:
I did this on purpose, because using the word dark triggers Material UI's built-in dark mode, which messes up the whole app. If you'd like, I can do defaultLight and customDark? @sadikkuzu

Reply:
Oh ok, i see. That makes sense. Thanks for clarification 🙏🏼

Well, current default&custom naming is ok 👍🏼 no need to change
Thanks again 🙏🏼✨🚀